app/controllers/Application.java

```diff
@@ -430,6 +430,7 @@ public static Result flowHistory() {
     if (results.size() == 0) {
       return notFound("Unable to find record on flow url: " + flowDefId);
     }
+    IdUrlPair flowDefPair = new IdUrlPair(flowDefId, results.get(0).flowDefUrl);
```

Comment:
Can you add some comments on why this has to be a pair ? Is flowdefid not unique enough for airflow scheduler ?